challenges/classes.js

```diff
@@ -1,7 +1,59 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
+/* ===== Prototype Practice ===== */
 
+// Task: You are to build a cuboid maker that can return values for a cuboid's volume or surface area. Cuboids are similar to cubes but do not have even sides. Follow the steps in order to accomplish this challenge.
+
+/* == Step 1: Base Constructor ==
+  Create a constructor function named CuboidMaker that accepts properties for length, width, and height
+*/
+
+
+class CuboidMaker {
+    constructor(maker){
+        this.length = maker.length;
+        this.width = maker.width;
+        this.height = maker.height;
+    }
+  }
+  
+  /* == Step 2: Volume Method ==
```

Comment:
comments like these will help you think things through if you get stuck and will help you understand what you were doing when you take a look back